scripts/SasDeltaEvidenceCache.psm1

```
@@ -0,0 +1,162 @@
+            EvidenceStrengthTier = if ($tiers.Count -gt 0) { $tiers[0] } else { 'NONE' }
+            SerialIdentityConfirmed = $identityConfirmed
+            ReachabilityStatus = $reachability
+            OpenPorts = $openPorts
+            ResolvedAddress = [string]$first.resolved_address
+            MacAddress = [string]$first.mac_address
+            ADCandidateStatus = [string]$first.ad_candidate_status
+            TrackerStatus = [string]$first.tracker_status
+            EvidenceType = [string]$first.evidence_type
+        }
+    }
+    return @($snapshots)
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'ConvertFrom-SasEvidenceArtifactPackages' returns an object of type 'System.Object[]' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)